quartz/components/scripts/util.ts

```diff
@@ -24,3 +24,16 @@ export function removeAllChildren(node: HTMLElement) {
     node.removeChild(node.firstChild)
   }
 }
+
+const canonicalRegex = /<link rel="canonical" href="([^"]*)"/
```

Comment:
can you comment on the caes where this run into?

I assume this has to do with files emitted by AliasRedirect?

Reply:
Yes, this specifically handles the HTML redirects emitted by AliasRedirect.

Sure I'll add a comment a bit later